common/src/main/java/satisfy/bloomingnature/registry/ObjectRegistry.java

```diff
@@ -326,28 +326,6 @@ static Item.Properties getSettings() {
     public static ResourceKey<ConfiguredFeature<?, ?>> configuredFeatureKey(String name) {
         return ResourceKey.create(Registries.CONFIGURED_FEATURE, new ResourceLocation(BloomingNature.MOD_ID, name));
     }
-    
-    public static void commonInit() {
```

Comment:
This function wasn't getting called anywhere and had duplicates in it. It also didn't account for every burnable wood item in the mod.

Reply:
Looking deeper into it, it seems like the tag system isn't picking up on the wood types which is why all of these furnace recipes weren't added. I don't know enough to know why the tags aren't working though. Probably better to fix that underlying issue instead of my change